hpat/tests/test_series.py

```diff
@@ -1756,13 +1756,129 @@ def test_impl(S, param_skipna):
             result = hpat_func(S, param_skipna)
             self.assertEqual(result, result_ref)
 
-    def test_series_value_counts(self):
+    def test_series_value_counts_numbers(self):
         def test_impl(S):
             return S.value_counts()
+
+        data_to_test = [[1, 2, 3, 1, 1, 3],
+                        [1, 2, 3, np.nan, 1, 3, np.nan, np.inf],
+                        [0.1, 3., np.nan, 3., 0.1, 3., np.nan, np.inf]]
+
         hpat_func = hpat.jit(test_impl)
 
-        S = pd.Series(['AA', 'BB', 'C', 'AA', 'C', 'AA'])
-        pd.testing.assert_series_equal(hpat_func(S), test_impl(S))
+        for data in data_to_test:
+            S = pd.Series(data)
+            pd.testing.assert_series_equal(hpat_func(S), test_impl(S))
+
+    def test_series_value_counts_numbers_all_unique(self):
+        ''' Pandas and SDC results are slightly different because of sorting unique values '''
```

Comment:
[pep8 recommendation](https://www.python.org/dev/peps/pep-0008/#string-quotes): `For triple-quoted strings, always use double quote characters to be consistent with the docstring convention in PEP 257.`